src/probabilit/distributions.py

```diff
@@ -106,22 +106,31 @@ def from_log_params(cls, mu, sigma):
         return Distribution("lognorm", s=sigma, scale=Exp(mu))
 
 
-def PERT(minimum, mode, maximum, gamma=4.0):
+def PERT(low, mode, high, low_perc=0.1, high_perc=0.9, gamma=4.0):
     """Returns a Beta distribution, parameterized by the PERT parameters.
 
     A high gamma value means a more concentrated distribution.
 
     Examples
     --------
-    >>> PERT(0, 6, 10)
+    >>> PERT(2,5,7)
+    Distribution("beta", a=3.50..., b=2.49..., loc=-1.29..., scale=10.04...)
+    >>> PERT(0, 6, 10, low_perc=0, high_perc=1)
     Distribution("beta", a=3.4, b=2.6, loc=0, scale=10)
-    >>> PERT(0, 6, 10, gamma=10)
-    Distribution("beta", a=7.0, b=5.0, loc=0, scale=10)
     """
     # Based on Wikipedia and another implementation:
     # https://en.wikipedia.org/wiki/PERT_distribution
-    # https://github.com/Calvinxc1/PertDist/blob/6577394265f57153441b5908147d94115b9edeed/pert/pert.py#L80
-    a, b, loc, scale = _pert_to_beta(minimum, mode, maximum, gamma=gamma)
+    if not ((0 <= low_perc <= 1.0) and (0 <= high_perc <= 1.0)):
```

Comment:
perhaps check low_perc < high_perc.

also, you removed a link that was a bit nice to have. add back unless you had a reason?